bin/solve-orient.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+system_file=$1
+input_file=$2
+output_file=$3
+
+mkdir -p $output_file && rmdir $output_file # Dirty hack to ensure output_file directory exists.
+
+orient/bin/orient solve --system=$system_file --in=$input_file | jq > $output_file
```

Comment:
`jq` is not a dep-- should probably add it to `bin/install-deps-orient.sh` and `bin/install-deps-orient-user.sh`?